docker/Dockerfile.xenial_debug

```diff
@@ -26,3 +27,11 @@ RUN apt-get update && \
    && apt-get upgrade -y \
    && apt-get clean \
    && rm -rf  /var/log/*log /var/lib/apt/lists/* /var/log/apt/* /var/lib/dpkg/*-old /var/cache/debconf/*-old
+
+# upgrade libstdc++6
+# hadolint ignore=DL3008
+RUN add-apt-repository ppa:ubuntu-toolchain-r/test \
+ && apt-get update \
+ && apt-get install --no-install-recommends -y --only-upgrade libstdc++6 \
```

Comment:
prowbazel installs `libc++-8-dev` -- is it ok this is different?

Reply:
It's ok. After upgrading libstdc++6, GLIBCXX_3.4.22 is supported.

Reply:
@howardjohn that's different library (libc++ vs libstdc++).